tests/queries/0_stateless/01259_combinator_distinct.sql

```diff
@@ -12,3 +12,11 @@ EXPLAIN SYNTAX SELECT countIf(DISTINCT number % 10, number % 5 = 2) FROM numbers
 
 SELECT sumIf(DISTINCT number % 10, number % 5 = 2) FROM numbers(10000);
 EXPLAIN SYNTAX SELECT sumIf(DISTINCT number % 10, number % 5 = 2) FROM numbers(10000);
+
+SET optimize_countdistinctif_rewrite = FALSE;
+SELECT countDistinctIf(number % 10, number % 5 = 2) FROM numbers_mt(100000000);
+EXPLAIN SYNTAX SELECT countDistinctIf(number % 10, number % 5 = 2) FROM numbers_mt(100000000);
+
+SET optimize_countdistinctif_rewrite = TRUE;
+SELECT countDistinctIf(number % 10, number % 5 = 2) FROM numbers_mt(100000000);
+EXPLAIN SYNTAX SELECT countDistinctIf(number % 10, number % 5 = 2) FROM numbers_mt(100000000);
```

Comment:
Need to add new line.

Reply:
@ucasfl it is standard, and also a lot of text processing utilities expect for file to have new line at the end.